Pod/Classes/BONText.m

```diff
@@ -311,6 +311,13 @@ - (BONStringDict *)attributes
         populateParagraphStyleIfNecessary();
         paragraphStyle.lineSpacing = self.lineSpacing;
     }
+    
+    // Line Break Mode
+    
+    if (self.lineBreakMode != NSLineBreakByWordWrapping) {
```

Comment:
I would prefer not to rely on the implementation detail that this property is initialized to `0`, which happens to equal `NSLineBreakByWordWrapping`. Please add that to `-init` of `BONText.m`.